src/components/top-navbar/index.js

```diff
@@ -0,0 +1,19 @@
+import { onNavigate } from '../../utils/history.js';
+import { signOut } from '../../services/index.js';
+
+export const TopNavbar = () => {
+  const topNavbar = document.createElement('nav');
```

Comment:
```suggestion
  const topHeader = document.createElement('header');
```
Acho que esse componente ao invés de ser uma `nav` poderia ser um `header`. Semanticamente é o que acho que faz mais sentido, de acordo com o que li no [MDN](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header).